src/test/scala/strawman/collection/test/Test.scala

```diff
@@ -288,6 +288,12 @@ class StrawmanTest {
     println(xs17.to(List))
   }
 
+  def sortedSets(xs: strawman.collection.immutable.SortedSet[Int]): Unit = {
+    val xs1 = xs.map((x: Int) => x.toString)
+    val xs2: SortedSet[String] = xs1
+    val xs3: strawman.collection.immutable.Set[String] = xs.map((x: Int) => x.toString)
```

Comment:
Looks like a type inferencer bug. This is as far as I got: https://issues.scala-lang.org/browse/SI-10194